paddle/fluid/operators/digamma_op.cu

```diff
@@ -0,0 +1,64 @@
+/* Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include <unsupported/Eigen/SpecialFunctions>
+#include "paddle/fluid/operators/digamma_op.h"
+#include "paddle/fluid/operators/elementwise/elementwise_op_impl.cu.h"
+#include "paddle/fluid/operators/math/complex_functors.h"
+
+namespace paddle {
+namespace operators {
+
+template <typename T, typename Enable = void>
+struct CudaDigammaFunctor;
+
+template <typename T>
+struct CudaDigammaFunctor<T, math::NoComplex<T, math::Real<T>>> {
+  __device__ __forceinline__ T operator()(const T* args) const {
+    return Eigen::numext::digamma(args[0]);
+  }
+};
+
+template <typename T>
+class DigammaKernel<platform::CUDADeviceContext, T>
```

Comment:
here is faster? maybe can test is and and some comments

Reply:
it isn't faster in the test, so removed it.